CharlieBackend.Api/Middlewares/IsAccountActiveMiddleware.cs

```diff
@@ -8,7 +10,7 @@
 namespace CharlieBackend.Api.Middlewares
 {
     /// <summary>
-    /// 
+    /// A helper class that determines whether the user is active or not
```

Comment:
```suggestion
    /// Middleware that restricts requests from users with deactivated accounts.
```
Small edit to summary comment that, IMO, describes the purpose of this middleware component a little bit better.